packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -1960,7 +1960,9 @@ module.exports = function (log, config) {
     const action = gettext('Create a password');
     const translatorParams = { productName };
     const subject = translator.format(
-      translator.gettext('%(productName)s payment confirmed'),
+      translator.gettext(
+        'Welcome to %(productName)s: Please set your password.'
```

Comment:
Since we don't use Fluent for emails (yet), how do I ensure that this updated string will get localized? I see where we pull in localization files using Jed, but I don't see how that gets pushed to `fxa-content-server-l10n`.

Reply:
Ah I found part of it; the fxa-content-server-l10n repo pulls from various fxa packages, including the auth-server: https://github.com/mozilla/fxa-content-server-l10n/blob/master/scripts/extract_strings.sh and https://github.com/mozilla/fxa-content-server-l10n/blob/master/scripts/extract-and-pull-request.sh .